Invocation_DetectHeartbeatTimeoutTest.whenCallTimeoutExpired_ButOperationHeartbeatHasNot

```
java.lang.AssertionError: expected:<NO_TIMEOUT__HEARTBEAT_TIMEOUT_NOT_EXPIRED> but was:<TIMEOUT>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_DetectHeartbeatTimeoutTest$1.run(Invocation_DetectHeartbeatTimeoutTest.java:134)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:875)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:889)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_DetectHeartbeatTimeoutTest.assertDetectHeartbeatTimeoutEventually(Invocation_DetectHeartbeatTimeoutTest.java:131)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_DetectHeartbeatTimeoutTest.whenCallTimeoutExpired_ButOperationHeartbeatHasNot(Invocation_DetectHeartbeatTimeoutTest.java:101)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-OracleJDK1.6/com.hazelcast$hazelcast/920/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/Invocation_DetectHeartbeatTimeoutTest/whenCallTimeoutExpired_ButOperationHeartbeatHasNot/

--- a/hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/Invocation_DetectHeartbeatTimeoutTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/Invocation_DetectHeartbeatTimeoutTest.java
@@ -88,15 +88,15 @@ public class Invocation_DetectHeartbeatTimeoutTest extends HazelcastTestSupport 
     @Test
     public void whenCallTimeoutExpired_ButOperationHeartbeatHasNot() {
         Config config = new Config();
-        config.setProperty(OPERATION_CALL_TIMEOUT_MILLIS.getName(), "1000");
+        config.setProperty(OPERATION_CALL_TIMEOUT_MILLIS.getName(), "5000");
 
         TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(2);
         HazelcastInstance local = factory.newHazelcastInstance(config);
         HazelcastInstance remote = factory.newHazelcastInstance(config);
 
         OperationService opService = getOperationService(local);
-        Operation operation = new SlowOperation(SECONDS.toMillis(60));
-        InvocationFuture f = (InvocationFuture) opService.invokeOnPartition(null, operation, getPartitionId(remote));
+        InvocationFuture f = (InvocationFuture) opService.invokeOnPartition(new SlowOperation(SECONDS.toMillis(60))
+                .setPartitionId(getPartitionId(remote)));
 
         assertDetectHeartbeatTimeoutEventually(f.invocation, NO_TIMEOUT__HEARTBEAT_TIMEOUT_NOT_EXPIRED);
     }

